src/validations/invoice-validation.ts

```diff
@@ -0,0 +1,87 @@
+import { Invoice } from '../classes/invoice';
+import * as invoiceServices from '../services/invoice-service';
+import * as customerServices from '../services/customer-service';
+import * as supplierServices from '../services/supplier-service';
+
+const errorList = {
+    customerExists: `Customer does not exist!`,
+    supplierExist: `Supplier does not exist!`,
+    paymentStatus: `Invoice status is not valid!`,
+    invoicePeriode: `Period of the invoice is not valid!`,
+    creationDate: `Creation date is not valid!`,
+    startDate: `Start date is not valid!`
+}
+
+const unpaidStatus = 1;
+
+const checkCustomerExists = async ( id: number): Promise<boolean> => {
```

Comment:
Like this. (So function names are inconsistent).
Just return the check, also not a promise. This applies to all other cases.